packages/gatsby-theme-blog-core/src/components/posts.js

```diff
@@ -1,3 +1,14 @@
 import React from "react"
 
-export default props => <pre>{JSON.stringify(props.data, null, 2)}</pre>
+import PostList from "./post-list"
+
+const Posts = ({ data }) => { 
+    const posts = data.allBlogPost.edges
+    return (
+        <main>
```

Comment:
Side note: we should make sure prettier & lint-staged are set up in this monorepo cc @johno